src/store/soraCard/actions.ts

```diff
@@ -52,6 +56,15 @@ const actions = defineActions({
       commit.resetTotalXorBalanceUpdates();
     }, 1000 * 60 * 5);
   },
```

Comment:
Why do we need timeout for unsubscription?

Reply:
previously it was redirected to bridge view to prompt user to deposit. (unsubscription takes place when user leaves current view and goes to bridge). I thought it would be nice to save same subscription after user done with bridge tx and go back to see their balance. 
Probably unnecessary thing, got to be removed.